Automattic/vip
# Update/docs pt vip config

## Description

Updated the voice, tense, and format of command descriptions, examples, and usage models for the `vip config` command group to follow the VIP-CLI style guide.

## Pull request checklist


## New release checklist


## Steps to Test

Observe the results of the `--help` display for impacted commands:

* `vip config`
  * `vip config software`
    * `vip config software update`
    * `vip config software get`
  * `vip config envvar`
    * `vip config envvar set`
    * `vip config envvar get`
    * `vip config envvar delete`
    * `vip config envvar list`
    * `vip config envvar get-all`
